docs/pull_request_template.md

```diff
@@ -3,6 +3,9 @@ For this assignment all pull requests going  forward should contain the followin
 # Title
   - A ledgible title that briefly describes the contents of the pull request.
 
+## linting
+  - does you code lint?
```

Comment:
Is this supposed to say:
```suggestion
  - Does your code lint?
```

Reply:
Also, is Linting meant to have a capital L in the beginning?

Reply:
fixed